site/_posts/2023-07-11-get-more-value-out-of-fusionauth.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: blog-post
+title: Get More Value Out of FusionAuth Using the APIs for User Maintenance
+description: Customizing the order for your favorite food can make a good meal better. Customizing your end-users experience with user maintenance can make their experience with your application even better.
+author: Mark Robustelli
+image: blogs/get-more-value/get-more-value-header.png
+category: article
+tags: api user maintenance windows forms c#
+excerpt_separator: "<!--more-->"
+---
+
+So, you have just set up FusionAuth and are excited to take your new [customer identity and access management](/articles/ciam/) platform out for a spin. You spend some time setting up your Users, Applications, Tenants, etc. in the robust front end provided. Things are looking great. You have integrated your applications and are humming along. Now comes a real test, user maintenance.
+
+<!--more-->
+
+<br>
+
+## Introduction
+*“A new user with extra pickles, please.”*
+
+Customizing the order for your favorite food can make a good meal better. Customizing your end-users' experience with user maintenance can make their experience with your application even better.
+
+For any [secure application](/articles/security/steps-secure-your-authentication-system), user maintenance is a must. As developers, we can often overlook the time and effort it takes to maintain the users access and permissions for the shiny new application we created. In an ideal world, we do not want to be the ones to maintain the users. We want to give the task of user maintenance to the power users of the applications by making them application administrators. While those power users may be great at using the application, you don’t necessarily want to make them expert users of the FusionAuth front end. Well, we at FusionAuth would think it would be great if everyone was a master with our front end, but you may not share that same vision.
+
+We can harness the full power of FusionAuth in our application, hand off the user maintenance to the power users, and make it seamless for them. The application will need to ensure the user enters the correct information (**Validation**), guide them through setting up the new user correctly (**Workflow**), and we want to make the process feel like it is part of our application (**Branding**). We can accomplish all these through the use of the FusionAuth APIs.
```

Comment:
Do you mean something like "The application will need to ensure the **end** user enters the correct information" or like "The application will need to ensure **that Joe User** enters the correct information"

Reply:
The second. You could even set up the story a bit above in line with your quotes. "Imagine you are selling a virtual cash register application. You have a customer, Joe Lettuce, who you want to add to your system, since Joe just bought your app for his restaurant 'Joe's salads'. You need to make sure your employee, Susie Carrot, enters Joes information correctly and quickly. "....etc etc.

Reply:
love this suggestion, added.  let me know it works for you.